docs/help/getting-started/index.md

```diff
@@ -1,56 +1,11 @@
-# Getting Started
+# Getting started with Bump.sh
 
-Bump.sh connects people with APIs. From your API specifications, Bump.sh generates the documentation you need and offers many options to make it your custom and carefully crafted API experience.
+Bump.sh connects people with REST and event-driven APIs, by centralizing a reference point: their documentation, highlighted with the latest changes and updates. Customize their look and feel, manage who can access (read or maintain) which doc.
 
-## Create your first documentation
+From OpenAPI and AsyncAPI specifications, Bump.sh generates the documentation you need and offers many options to make it your custom and carefully crafted API experience. Those specifications are also supported by a wide variety of frameworks in various programming languages, which generate contract files automatically. 
```

Comment:
A bit too much marketing. This is a help center article, we do not need to push the product that much.